libs/emblem/src/components/market-emblem.tsx

```diff
@@ -0,0 +1,57 @@
+import { URL_BASE } from '../config';
+import { EmblemBase } from './emblem-base';
+import { useMarketInfo } from './hooks/use-market-info';
+import { getVegaChain } from './lib/get-chain';
+
+export type EmblemByMarketProps = {
+  market: string;
+  vegaChain?: string;
+  showBase?: boolean;
+  showQuote?: boolean;
+  contract?: never;
+  asset?: never;
+  wrapperClass?: string;
+};
+
+/**
+ * Given a Vega Market ID...
+ *
+ * @param market string the market ID
+ * @param vegaChain string the vega chain ID (default: Vega Mainnet)
+ * @returns React.Node
+ */
+export function EmblemByMarket(props: EmblemByMarketProps) {
+  const chain = getVegaChain(props.vegaChain);
+  const data = useMarketInfo(chain, props.market);
+  const showBase = props.showBase ?? true;
```

Comment:
non blocking: destructuring with a default property value tends to be more common place for default property values

```
export function EmblemByMarket({
  ...
  showBase = true,
  showQuote = true
}: EmblemByMarketProps) {
  ...
}
```

Reply:
I've used partial destructuring in a follow up. My reason for doing it this way was to pass props down to the underlying Emblem component directly, so they can also be controlled.

But now I'm not sure if that's actually a good idea or not. It feels a bit spaghetti like. I vote I just remove that ability. How does that feel?